docs/release-notes/release-notes-0.8.3.md

```diff
@@ -0,0 +1,253 @@
+
+`7bfc758` Use mempoolminfee instead of minrelaytxfee as tx feerate floor
+
+`bc2d604` Prevent absurdly low minsizes in yg-pe
+
+`aab14f6` fix taker fee estimation on sweep after maker responses
+
+`1d74222` Fix year 2038 problem in fidelity bond wallets
+
+`8639a96` When selecting utxos in yg, count in that change should be above dust threshold already on the first try
+
+`43cc960` taker: Account for off-by-one errors caused by rounding
+
+`9ea0a73` Hide "Total balance" in wallet-tool when only single mixdepth is displayed
+
+`f361c98` Better handling of non-standard tx'es, less extra output for csv and other small improvements
+
+### Qt UI improvements
+
+These are mostly fairly minor so grouped together:
+
+`2bde2f9` jm-qt: Don't annoy the user for removing an address
+
+`1c2b6cd` jm-qt: Clarify error message: "0" is a valid input to `numCPInput`
+
+`4599bd7` Fix autofreeze warning on Qt
+
+`7949374` Clear single send form after successful send
+
+`25b2e8e` Don't create logs, wallets and cmtdata subdirectories under scripts
+
+
+### Testing and dev related background fixes
+
+These are almost entirely not related to user functionality, but a couple of notes:
+
+Although `df83764` is minor, it relates to an important point: we cannot support importing keys of a different scriptPubKey type (e.g. native segwit BIP84) than the wallet being imported into. Key import is not something users should generally use, but if they do, it's very important to know this limitation.
+`03ea5bb` adds options to the `run_tests.sh` script.
+
+`41658aa` Remove socks.py, not used
+
+`adbbb24` address complaints of flake8 in jmclient
+
+`ab43961` Add flake8 linter script
+
+`512fab7` Fix remaining flake8 errors
+
+`fde23c3` Remove unused imports
+
+`4e4b15b` add tests cases for commitment sourcing
+
+`46281b5` Use python-bitcointx in fidelity bond wallet sync
+
+`392f51b` Remove rand_pow_array(), it's not used anymore
+
+`c30929a` Fix SNICKER client-daemon setup
+
+`eedd6e0` Ensure clients connect to correct daemon port
+
+`03ea5bb` Add command line option support and verbose output option
+
+`60e2bea` Move wallet_utils tests to test suite
+
+`df83764` Remove unused key-type param for importprivkey
+
+`c70f253` add test case for low feerate
+
+`79bb3ff` Allow to specify all conftest.py arguments to run_tests.sh
+
+`e7027ab` Change IRC ports for local tests
+
+`516e46a` Additional BIP32 test vector for hardened derivation with leading zeros
+
+`fb1d6cf` Remove unused function argument
+
+### Documentation updates
+
+`3edd07c` Remove reused donation addresses
+
+`16294bc` taker_utils: Fix typo in comment
+
+`46fe744` update payjoin comment in readme
+
+`f0cd0a9` Update SOURCING-COMMITMENTS.md
+
+`aa49169` add the community Telegram channel
+
+`b0ea821` small update to TODO list
+
+`e0cfae5` move IRC network from freenode to libera.chat (merges #884)
+
+`912b4d5` add missing whitespace to log
+
+`1125120` darkscience tor address v3
```

Comment:
This one should be mentioned somewhere above (not as just something testing / dev related, not important) too, it is recommended for existing users to change their darkscience onion address from v2 to v3 in `joinmarket.cfg`. Kinda important because v2 onion services will stop working soon.

Reply:
[darkscience Tor v2 Deprecation Timeline](https://darkscience.net/news/onion-v2-deprecation-timeline/)

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/912/commits/f1e440cd596e5d46d9da9d6bea0db9e1489d988d